services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/AlluUpdateService.kt

```diff
@@ -26,7 +26,7 @@ class AlluUpdateService(
         initialDelayString = "\${haitaton.allu.updateInitialDelayMilliSeconds}"
     )
     fun checkApplicationStatuses() {
-        logger.info("Trying to obtain lock $lockName to start uploading audit logs.")
+        logger.info("Trying to obtain lock $lockName to start checking Allu application histories.")
```

Comment:
Not related to your current task but this and other scheduled-related task are logging too much stuff at info level. This could be improved by either:
- using debug log
- removing happy path logs and focus on exceptional situations.

But yes, not related to this task so you can ignore this comment if you want.